src/main/java/com/example/teamboolean/apprentidash/AppUser.java

```diff
@@ -24,6 +24,18 @@
     @OneToMany (mappedBy = "user")
     List<Day> days;
 
+    public Day getCurrentday() {
```

Comment:
We probably want to move the getters and setters down to be with the other getters/setters